x/compute/internal/keeper/keeper.go

```diff
@@ -125,14 +125,15 @@ func (k Keeper) setParams(ctx sdk.Context, ps types.Params) {
 
 // Create uploads and compiles a WASM contract, returning a short identifier for the contract
 func (k Keeper) Create(ctx sdk.Context, creator sdk.AccAddress, wasmCode []byte, source string, builder string) (codeID uint64, err error) {
-	return k.create(ctx, creator, wasmCode, source, builder, &types.AccessConfig{Type: types.Everybody} /* , k.authZPolicy */)
-}
-
-func (k Keeper) create(ctx sdk.Context, creator sdk.AccAddress, wasmCode []byte, source string, builder string, instantiateAccess *types.AccessConfig /* , authZ AuthorizationPolicy */) (codeID uint64, err error) {
 	/*
-		if !authZ.CanCreateCode(k.getUploadAccessConfig(ctx), creator) {
-			return 0, sdkerrors.Wrap(sdkerrors.ErrUnauthorized, "can not create code")
-		}
+			   	return k.create(ctx, creator, wasmCode, source, builder, &types.AccessConfig{Type: types.Everybody}  , k.authZPolicy )
+			   }
+
+			   func (k Keeper) create(ctx sdk.Context, creator sdk.AccAddress, wasmCode []byte, source string, builder string, instantiateAccess *types.AccessConfig ) (codeID uint64, err error) {
+		/*
+			   		if !authZ.CanCreateCode(k.getUploadAccessConfig(ctx), creator) {
+			   			return 0, sdkerrors.Wrap(sdkerrors.ErrUnauthorized, "can not create code")
+			   		}
 	*/
```

Comment:
Adding so much whitespace is only going to make merges harder :scream: 
You want to just add a `/*` on its own line before the content, and a `*/` on its own line after the content...

Reply:
or just delete this stuff since we don't care what they change in wasmd 😛